keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -89,8 +88,8 @@ class ClusterHandler(
         .map { diff ->
           val spec = diff.desired
           val job = when {
-            diff.isCapacityOnly() -> spec.resizeServerGroupJob(resource.serviceAccount)
-            else -> spec.createServerGroupJob(resource.serviceAccount)
+            diff.isCapacityOnly() -> diff.resizeServerGroupJob()
+            else -> diff.createServerGroupJob()
```

Comment:
service account was being passed in here so that we could look up the current server group name from CloudDriver. That is now on the `current` property of the diff (we just looked it up milliseconds before so doing it again seems a waste).